tcbuilder/cli/build.py

```diff
@@ -384,21 +394,17 @@ def handle_easy_installer_output(props, storage_dir, union_params):
         handle_provisioning(output_dir, props.get("provisioning"))
 
 
-def handle_bundle_output(image_dir, storage_dir, bundle_props, tezi_props):
-    """Handle the bundle and combine steps of the output generation."""
+def handle_bundle_common(
+        storage_dir,
+        bundle_props,
+        is_compressed_tar=True
+    ):
+    """Handle the common steps of the bundle and combine for output generation."""
 
     if "dir" in bundle_props:
-        # Do a combine "in place" to avoid creating another directory.
-        combine_params = {
-            "image_dir": image_dir,
-            "bundle_dir": bundle_props["dir"],
-            "output_directory": None,
-            "tezi_props": translate_tezi_props(tezi_props),
-            "force": True
-        }
-        comb_be.combine_tezi_image(**combine_params)
+        return bundle_props["dir"]
```

Comment:
Since this common function handles the case where the user just provides a pre-made bundle directory, shouldn't the schema for raw-images also support this then? With the current changes the schema would only support a provided compose file and not bundle directory for raw images.

Reply:
yeah, this is exactly the idea, for now, only to support the `bundle` `compose-file`. This could be a `TODO` or if you guys think that should be added now, let me know.

Reply:
Okay well, I don't have a super strong opinion on whether it should be done now or later. I just wanted to check if it was an intended decision on your end. 

Out of curiosity is there still some extra work or something needed still to add support for the bundle directory for raw images in the `build` command?